graphql-cli/src/test/resources/serviceGen/expectedServices/Ballerina.toml

```diff
@@ -0,0 +1,4 @@
+[package]
+org = "graphql_test"
+name = "test_package"
+version = "0.1.0"
```

Comment:
Do we need to commit the `toml` file?

Reply:
In some tests, generated files are stored to a temp directory. This `.toml` file is copied there. So the directory can be checked for any compilation errors.